src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = System.lineSeparator();
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
do not use such naming(stringBuilder), how to guess what it stores?
```suggestion
        StringBuilder data = new StringBuilder();
```